pkg/noderesourcetopology/filter_test.go

```diff
@@ -586,6 +586,23 @@ func TestNodeResourceTopologyMultiContainerPodScope(t *testing.T) {
 	}
 }
 
+// should be filled by the user
+type testUserEntry struct {
+	// description contains the test type and tier as described in TEST.md
+	// and a short description of the test itself
+	description string
+	initCntReq  []map[string]string
+	cntReq      []map[string]string
+	statusErr   string
```

Comment:
I think it should be obvious and explicit the nodes and the NRTs being used here

Reply:
We're always using the same nodes and NRTs object, so why would we need them as variables?

Reply:
because making that implicit makes harder to understand at glance what the test is supposed to do. One has to lookup the implicit context instead of having it clearly spelled out.